frontend/pages/admin/IntegrationsPage/cards/Calendars/_styles.scss

```
@@ -36,12 +32,18 @@
     font-size: $x-small;
   }
 
+  &__save-changes,
+  &__oauth-scopes {
+    @include vertical-form-layout;
+    padding-bottom: $pad-large;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)